tests/test_activation_statistics.py

```diff
@@ -0,0 +1,82 @@
+import unittest
```

Comment:
nit: we should add the Copyright header to all python modules. See other python modules :)